Makefile

```diff
@@ -32,10 +39,10 @@ deploy-init:
 
 deploy-appbundle:
 	cp \
-		$(BUILD_DIR)/app.bundle.min.js \
-		$(BUILD_DIR)/app.bundle.min.map \
-		$(BUILD_DIR)/external_api.min.js \
-		$(BUILD_DIR)/external_api.min.map \
+		$(BUILD_DIR)/app.bundle.js \
+		$(BUILD_DIR)/app.bundle.map \
+		$(BUILD_DIR)/external_api.js \
+		$(BUILD_DIR)/external_api.map \
 		$(OUTPUT_DIR)/analytics.js \
```

Comment:
Lyubo just told what's the story behind this. And that those files are still minimized. We need to further discuss it.